drivers/sensor/mcp9600/Kconfig

```diff
@@ -0,0 +1,14 @@
+# I2C Compatible, Thermocouple EMF to Temperature Converter
+#
+# Copyright (c) 2022, Maxmillion McLaughlin
+# Copyright (c) 2020, SER Consulting LLC / Steven Riedl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config MCP9600
+	bool "MCP9600 i2c thermocouple temperature sensor"
+	default y
+	depends on DT_HAS_MICROCHIP_MCP9600_ENABLED
+	depends on I2C
```

Comment:
`depends on I2C` should now be `select I2C`

Reply:
should be resolved